app/src/main/kotlin/com/softteco/template/ui/feature/settings/SettingsViewModel.kt

```diff
@@ -22,7 +22,7 @@ class SettingsViewModel @Inject constructor(
 
```

Comment:
Here as well, 
`  @Named("themeMode") private val themeModeDataStore: DataStore<Preferences>,`
could be changed to 
`private val dataStore: DataStore<Preferences>,`